src/forms/FormFieldAdvancedSelection/AdvancedSelectionDrawer.tsx

```diff
@@ -437,7 +436,7 @@ const AdvancedSelectionDrawer = (props: AdvanceSelectionDrawerPropTypes): ReactE
 				type='drawer'
 				state={state}
 				dispatch={dispatch}
-				fields={fields}
+				fields={fields as FieldDef[]}

```

Comment:
For consistency please move the "as FieldDef[]"s next to their respective "const fields"